### DISCUSSION TITLE
DISM support

### DISCUSSION THREAD

**Discussion by c-michail** [diversenok/NtUtilsLibrary#7] [General]
Hello,
  Do you plan to include dismapi.h headers ?
  
  Thank you

[dismapi.zip](https://github.com/user-attachments/files/15937450/dismapi.zip)

**Comment by diversenok** [comment]
Hi! Thanks for the idea. I didn't realize DISM definitions were available in ADK. I added them into [Ntapi.dismapi.pas](https://github.com/diversenok/NtUtilsLibrary/blob/master/Headers/Ntapi.dismapi.pas) plus provided wrappers in [NtUtils.Dism.pas](https://github.com/diversenok/NtUtilsLibrary/blob/master/NtUtils.Dism.pas).

**Reply by c-michail** [reply]
Thank you
